package.json

```diff
@@ -32,7 +32,7 @@
     "@internetarchive/infinite-scroller": "1.0.1",
     "@internetarchive/local-cache": "^0.2.1",
```

Comment:
q: i noticed this was deleted in certain files, can we remove here?

Reply:
Possibly! I'll double-check that there isn't anything else using this package...

Reply:
Here is ticket for this: https://webarchive.jira.com/browse/WEBDEV-6595